### PR TITLE
+ Added Embedded Systems Topic

### PR DESCRIPTION
**Pull request by kamatajinkya**
+ Added Resources for Beginner and Intermediate Level
+ Added generals tips for embedded
+ Updated the list of topics

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Please click on following link to see the more info about the specific topic
 * [Computer Vision](computer-vision)
 * [Control Theory](control-theory)
 * [Deep Learning](deep-learning)
+* [Embedded](embedded)
 * [Internet of Things-IoT](iot)
```
Comment (take2rohit) [inline README.md:26]: I think it should be `Embedded Systems`

File: embedded-systems/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* [C Programming For Microcontrollers Featuring ATMEL's Butterfly and the Free WinAVR Complier](http://dsp-book.narod.ru/CPMicro.pdf)
+**_Why this book?:_**
+This book starts from the absolute basics. It presents C programming with an assumption that you are completely new to the programming scene, while simultenously ensuring a healthy level to technical recourse. It is a quick and easy read, with a lot of code and project examples. **_What to keep in min?:_**
+Both the butterfly board and the WinAVR compilers are outdated and suitable substitutes might be required to follow the book properly.
+**_Estimated Time to complete:_**
+20 hours
+
+* [Tiva Launchpad Workshop](https://processors.wiki.ti.com/index.php/Getting_Started_with_the_TIVA%e2%84%a2_C-Series_TM4C123G_LaunchPad?DCMP=tivac&HQS=TM4C123G-Launchpad-Workshop)
+**_Why this works?:_**
```
Comment (akshaykulkarni07) [inline embedded-systems/README.md]: Why this workshop? typo

**Review by take2rohit — CHANGES_REQUESTED** [review]
Please review these changes. Thanks

File: embedded-systems/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Embedded Systems
+
+## Resources
+
+### Beginner Level
+
+* [C Programming For Microcontrollers Featuring ATMEL's Butterfly and the Free WinAVR Complier](http://dsp-book.narod.ru/CPMicro.pdf)
+**_Why this book?:_**
+This book starts from the absolute basics. It presents C programming with an assumption that you are completely new to the programming scene, while simultenously ensuring a healthy level to technical recourse. It is a quick and easy read, with a lot of code and project examples. **_What to keep in min?:_**
```
Comment (akshaykulkarni07) [inline embedded-systems/README.md]: what to keep in "mind" (typo)

File: README.md (hunk 1 above)
Comment (akshaykulkarni07) [reply README.md:26]: it has to match the folder name, so `embedded-systems` specifically

**Review by akshaykulkarni07 — COMMENTED** [review]
Some minor typos and inconsistencies (like micro controller v/s microcontroller and full stops not put every where consistently).

File: embedded-systems/README.md (hunk 2 above)
Comment (kamatajinkya) [reply embedded-systems/README.md:15]: Its not a typo. Why this workshop as in "Why is this workshop useful?"

**Comment by kamatajinkya** [comment]
Let me update this and push

**Comment by aditya-shirwatkar** [comment]
I don't know anything about embedded systems, I think you've assigned the wrong reviewer :grimacing:

**Comment by kamatajinkya** [comment]
> I don't know anything about embedded systems, I think you've assigned the wrong reviewer
@aditya-shirwatkar It not necessary that the reviewer must know about embedded systems. You can do a check on documentation rules and such

**Comment by kamatajinkya** [comment]
Updated the document. If there is a spelling or a grammar mistake it would be nice if u guys can use the suggest changes feature to correct that.

**Review by akshaykulkarni07 — APPROVED** [review]

**Review by aditya-shirwatkar — APPROVED** [review]

**Comment by kamatajinkya** [comment]
@GlazeDonuts @take2rohit Need ur inputs to merge the branch

**Review by take2rohit — APPROVED** [review]

**Pull request merged**